packages/fxa-content-server/server/lib/routes/payment.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const flowMetrics = require('../flow-metrics');
+const logger = require('../logging/log')('routes.index');
+const vhost = require('vhost');
```

Comment:
Seems like this is a leftover from something? Doesn't seem used anywhere and there's no `vhost` dependency in `package.json`